src/com/amazon/ion/benchmark/schema/constraints/Range.java

```diff
@@ -13,7 +13,7 @@
 // Processing the constraint value which contains 'range' annotation.
 public class Range {
     private static final String KEYWORD_RANGE = "range";
-    IonSequence sequence;
+    public final IonSequence sequence;
```

Comment:
I'd recommend making this `private` and adding an accessor method with the appropriate visibility. Or, if it's only used by subclasses, consider `protected`.

Reply:
> I'd recommend making this `private` and adding an accessor method with the appropriate visibility. Or, if it's only used by subclasses, consider `protected`.

Sounds good, in the next commit the accessor method will be added.